qa/automation/ScoringPortal.Tests.Scenarios/3. Sign In/SignIn.feature

```diff
@@ -0,0 +1,247 @@
+    And user clicks 'Next' button
+    Then user is signed in
+    And application webpage is oped
+    And "New User" name can be changed
+    And "Log Out" button is active
+    And "Drop your CSV.." button is active
+
+    Examples:
+        | browsers       |
+        | Chrome         |
+        | FireFox        |
+        | Microsoft Edge |
+
+@tc:3
+
+
+Scenario Outline: 3.1.1.1_04 Sign in with phone number of wrong format if the user is not logged in
+    Given user clicked on "Sign in with Google" button
+    And webpage with a textfield is opened
+    When User enters invalid phone number into textfield 
+    And User clicks 'Next' button
+    Then the error message appears that account has not been found
+
+    Examples:
+        | phone numner format                  |
+        | 8 digits                             |
+        | 9 digits and a letter                |
+        | 8 digits, 1 letter, 1 special symbol |
+@tc:4
+
+
+Scenario Outline: 3.1.1.1_05 Sign in  if the user is logged in the only google account
+    Given user is logged in the only google account in the browser
+    When User clicks on  "Sign in with Google" button
+    Then user is signed in
+    And application webpage is oped
+    And "New User" name can be changed
+    And "Log Out" button is active
+    And "Drop your CSV.." button is active
+    
+    Examples:
+        | browsers       |
+        | Chrome         |
+        | FireFox        |
+        | Microsoft Edge |
+@tc:5
+
+Scenario Outline: 3.1.1.1_06 Sign in with full email address if the user is logged in into 2 google accounts
+    Given user is logged into 2 google accounts in the browser
+    And webpage with a "Sign in with Google"
+    When user clicks on "Sign in with Google" button 
+    And User chooses the needed google account {email@google.com} from dropdown
+    Then user is signed in
+    And application webpage is oped
+    And "New User" name can be changed
+    And "Log Out" button is active
+    And "Drop your CSV.." button is active
+
+    Examples:
+        | browsers       |
+        | Chrome         |
+        | FireFox        |
+        | Microsoft Edge |
+@tc:6
+
+Scenario Outline: 3.1.1.1_07 Sign in if the user is logged in into google account and into the application in another browser
+    Given user is logged into google account in the browser A
+    And webpage with a "Sign in with Google"
+    And user is signed in the application with the same google account in the browser B
+    When user clicks on "Sign in with Google" button 
+    Then user is signed in
+    And application webpage is oped
+    And "New User" name can be changed
+    And "Log Out" button is active
+    And "Drop your CSV.." button is active
+
+    Examples:
+        | browsers       |
+        | Chrome         |
+        | FireFox        |
+        | Microsoft Edge |
+@tc:7
+
+Scenario Outline: 3.1.1.1_08 Sign in with google account if user is logged off with valid password
```

Comment:
Guess you meant 'logged out'

Reply:
Do you mean that the user have already signed in successfully at least once? Doesn't it mean that the login and password information should be filled automatically in this browser?